share/smack/svcomp/utils.py

```diff
@@ -42,6 +44,15 @@ def svcomp_frontend(args):
   smack.top.clang_frontend(args)
 
 def svcomp_process_file(args, name, ext):
+  # Check if property is vanilla reachability, and return unknown otherwise
+  if args.svcomp_property:
+    with open(args.svcomp_property, "r") as f:
+      prop = f.read()
+    if "valid-deref" in prop:
```

Comment:
So what will happen if the `svcomp_property` file does not exist or cannot be read?